charts/geoserver/Chart.yaml

```diff
@@ -10,7 +10,7 @@ description: An open source server for sharing geospatial data.
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 1.2.0
+version: 1.3.0
```

Comment:
My only concern is this. Maybe this should be 1.2.1 since it is just adding a single variable to the chart? @robkooper what do you think?